esp-hal/src/soc/esp32c2/efuse/fields.rs

```diff
@@ -0,0 +1,207 @@
+//! eFuse fields for the ESP32-C2.
+//!
+//! This file was automatically generated, please do not edit it manually!
```

Comment:
If it's possible, could we somehow get a git rev of where the csv that generated the file? Maybe it won't be that useful, but might save us some trouble in the future.